### ISSUE TITLE
Uncaught TypeError: undefined is not a function

### ISSUE THREAD

**Issue by mrmechko** [dundalek/atom-markdown-mindmap#3]
It seems to happen whenever I stop dragging.

**Atom Version**: 1.0.7
**System**: Mac OS X 10.10.5
**Thrown From**: [markdown-mindmap](https://github.com/dundalek/atom-markdown-mindmap) package, v0.2.2

### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /Users/mechko/.atom/packages/markdown-mindmap/lib/markdown-mindmap-view.coffee:279

TypeError: undefined is not a function
  at MarkdownMindmapView.module.exports.MarkdownMindmapView.copyToClipboard (/Users/mechko/.atom/packages/markdown-mindmap/lib/markdown-mindmap-view.coffee:279:6)
  at space-pen-div.atom.commands.add.core:copy (/Users/mechko/.atom/packages/markdown-mindmap/lib/markdown-mindmap-view.coffee:97:37)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:241:29)
  at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61)
  at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:156:19)
  at EventEmitter.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:70:30)
  at emitOne (events.js:77:13)
  at EventEmitter.emit (events.js:166:7)

```

### Commands

```
     -2:21.5.0 core:select-all (atom-text-editor.editor.is-focused)
     -2:21 core:paste (atom-text-editor.editor.is-focused)
     -2:17.1.0 core:save (atom-text-editor.editor.is-focused)
     -2:14.6.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -2:14.6.0 core:cancel (atom-text-editor.editor.is-focused)
     -2:14.1.0 settings-view:open (atom-text-editor.editor.is-focused)
     -2:07.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:29.3.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:26.9.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:26.9.0 markdown-mindmap:toggle (atom-text-editor.editor)
     -0:01.2.0 core:copy (div.markdown-mindmap.native-key-bindings)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-material-ui",
      "one-dark-syntax"
    ],
    "projectHome": "/Users/mechko/Projects"
  }
}
```

### Installed Packages

```coffee
# User
atom-material-ui, v0.6.0
language-lisp, v0.1.2
language-scala, v1.1.3
markdown-mindmap, v0.2.2
markdown-preview-plus, v1.7.0

# Dev
No dev packages
```

**Comment by dundalek** [comment]
Thanks for reporting, fixed in v0.2.3. There was extra code for handling commands. I disabled it until the commands are properly implemented.

**Comment by brianryer** [comment]
Trying to copy the mindmap html produced this error.
